bin/yaml/kotlin-native.yaml

```diff
@@ -30,3 +30,6 @@ compilers:
         url: https://github.com/JetBrains/kotlin/releases/download/v1.5.20/kotlin-native-linux-1.5.20.tar.gz
       - name: 1.5.21
         url: https://github.com/JetBrains/kotlin/releases/download/v1.5.21/kotlin-native-linux-1.5.21.tar.gz
+      - name: 1.5.30
+        url: https://github.com/JetBrains/kotlin/releases/download/v1.5.30/kotlin-native-linux-x86_64-1.5.30.tar.gz
+        configure_command: [mv, kotlin-native-linux-x86_64-1.5.30, kotlin-native-linux-1.5.30]
```

Comment:
Presumably something changed in this release, and you wanted the CE-side configuration to remain the same, so renamed it here? I thnk there's another way to achieve this; will merge and try and fix it up (this YAML stuff is not obvious, I'm sorry)

Reply:
Oh! I see there's a few other things that already do this :blush: - I'll post here if I find a better way but...this is awesome. thanks you!